docs/cli/logs.md

```diff
@@ -0,0 +1,50 @@
+# Function Logs
```

Comment:
I'd like to see it mentioned that logs work with standard tooling also i.e. `kubectl logs` and `docker service logs`

Reply:
I am not sure why we would want to highlight that in the docs for the CLI.  I would prefer to avoid trying to continue documenting that path, since the `faas-cli logs` is supposed to be the "official" way to do it that abstracts the various ways functions can be deployed in different providers.

Reply:
This reason: I don't want people to think that there is only a magic way to access logs as there is with something like Fission or OpenWhisk.

> Something similar to: OpenFaaS wraps the existing container-native logs and making it easier to access them through its CLI and API. They are still accessible via `kubectl logs` at the deployment-level.

Reply:
@alexellis i have updated the intro section in the last commit